### DISCUSSION TITLE
Form with Cancel and Submit buttons

### DISCUSSION THREAD

**Discussion by Estigy** [orchidsoftware/platform#2283] [Q&A]
Hi!

I want to add a group of buttons/links at the bottom of my forms: "Submit" sends the form, "Cancel" leads back to the list.
What's the best way to do this?

I'm totally new to Orchid, so my first idea was to group two buttons next to each other - but buttons don't have a route() method I would have used with the Cancel button.
The second option I saw was to use a Link as the Cancel button so I can use its route() method; but the two itemsare not being rendered on the same horizontal line - the link is being put about 15 px down.

Please advise :-)
Thanks!

**Comment by AlexanderWM** [comment]
```
Group::make([
         Button::make('Save')
           ->icon('note')
          ->method('createOrUpdate')
          ->type(Color::PRIMARY())
          ->canSee(!$this->post->exists),

         Button::make('Cancel')
          ->icon('arrow-left')
          ->method('toList')
          ->rawClick(),

          ])->autoWidth()->toEnd(),

```

**Reply by Estigy** [reply]
Hi, Alexander.

I had hoped there was a solution where I would not need to put an additional toList() method into every edit screen only to get back to the list. As I understand it, I would have to implement the "toList" Method from your example by myself, right?

**Reply by AlexanderWM** [reply]
Right.  Example:

```
    public function toList(Plan $plan)
    {
        return redirect()->route('platform.plan.list'); 
    }
```
Or you can simple do return redirect()->back();

**Reply by Estigy** [reply]
Thanks, Alexander.

I ended up putting this method together with a `getCancelButton()` method that returns a preconfigured button in a trait that I can use throughout my application :-)
